lib/griddler/amazon_s3_ses/adapter.rb

```diff
@@ -89,11 +90,11 @@ def multipart?
       end
 
       def text_part
-        force_body_to_utf_8_string(multipart? ? message.text_part.try(:body) : message.try(:body))
+        force_body_to_utf_8_string(multipart? ? message.text_part.try(:body) : Sanitize.clean(message.try(:body)))
       end
 
       def html_part
-        multipart? ? force_body_to_utf_8_string(message.html_part.try(:body)) : nil
+        force_body_to_utf_8_string(multipart? ? message.html_part.try(:body) : message.try(:body))
```

Comment:
Do you know if we have any logic downstream that removes javascript tags or any other dangerous tags?  Since we load in the sanitize class, I wonder if it is a good time to sanitize the HTML at the same time.  

If it is not a concern, then there is nothing to be done.

Reply:
We don't have any downstream sanitizing before storing in the database. However, we don't sanitize content being created using the UI either; we sanitize the content when displaying it. This was an intentional decision to only sanitize from the frontend (and be able to put pretty much anything in the db) because we wanted to be least surprising as possible--not changing something on a user unexpectedly.